container/attribute.go

```diff
@@ -4,75 +4,99 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/container"
 )
 
-type (
-	Attribute  container.Attribute
-	Attributes []Attribute
-)
-
-// NewAttribute creates and initializes blank Attribute.
+// Attribute represents v2-compatible container attribute.
```

Comment:
```suggestion
// Attribute represents container attribute.
```

Reply:
why? thought all public types should have such info